modules/programs/waybar.nix

```diff
@@ -282,7 +282,7 @@ in {
         })
       ];
 
-      home.packages = [ cfg.package ];
+      home.packages = if cfg.package != null then [ cfg.package ] else [ ];
```

Comment:
more elegant with `lib.optionals`

Reply:
@teto thank you, I hope I got it right, new to nix :)